File: executable/r/custom-size-subplot.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(1, 2, 3), 
  y = c(4, 5, 6), 
  type = "scatter"
)
trace2 <- list(
  x = c(20, 30, 40), 
  y = c(50, 60, 70), 
  xaxis = "x2", 
  yaxis = "y2", 
  type = "scatter"
)

layout <- list(
  xaxis = list(domain = c(0, 0.7)), 
  yaxis2 = list(anchor = "x2"), 
  xaxis2 = list(domain = c(0.8, 1))
)



response <- p$plotly(trace0, trace1, kwargs=list(layout=layout, filename="custom-size-subplot", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/python/custom_size_subplot.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')

trace1 = Scatter(
    x=[1, 2, 3],
    y=[4, 5, 6]
)
trace2 = Scatter(
    x=[20, 30, 40],
    y=[50, 60, 70],
    xaxis='x2',
    yaxis='y2'
)
data = Data([trace1, trace2])
layout = Layout(
    xaxis=XAxis(
        domain=[0, 0.7]
    ),
    xaxis2=XAxis(
        domain=[0.8, 1]
    ),
    yaxis2=YAxis(
        anchor='x2'
    )
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, auto_open=False, filename='custom-size-subplot')